exchanges/deribit/deribit_wrapper.go

```diff
@@ -591,8 +591,15 @@ func (d *Deribit) SubmitOrder(ctx context.Context, s *order.Submit) (*order.Subm
 	if err != nil {
 		return nil, err
 	}
-	timeInForce := ""
-	if s.ImmediateOrCancel {
+	var timeInForce string
+	switch s.TimeInForce {
```

Comment:
Use method `Is` as we are flagging and can be  `order.GoodTillDay | order.PostOnly` for example.